### PR TITLE
Feature/edx 10

### PR DESCRIPTION
**Pull request by cditcher**

### CODE REVIEW

File: test/src/test_cases/edx-schools-frontend/test-school-inbox.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ fixture `school-inbox`
        getToken().then(async (data) => {
             token = data.access_token;
             testExchange = await createSecureExchange(token, JSON.stringify(testExchange));
+            log.info("Secure exchange created with id: " + testExchange.secureExchangeID);
```
Comment (arcshiftsolutions) [inline test/src/test_cases/edx-schools-frontend/test-school-inbox.js]: Please remove

File: frontend/src/components/SecureExchange/AccessUsersPage.vue [hunk 2]
```diff
@@ -72,6 +72,7 @@ export default {
   },
   created() {
     this.getUsersData();
+    console.log('USER INFO: ' + JSON.stringify(this.userInfo, null, 3));
```
Comment (arcshiftsolutions) [inline frontend/src/components/SecureExchange/AccessUsersPage.vue]: Please remove

File: test/src/test_cases/edx-schools-frontend/test-school-inbox.js (hunk 1 above)
Comment (cditcher) [reply test/src/test_cases/edx-schools-frontend/test-school-inbox.js]: Ummm weird. This code is not on my branch, not sure how it appearing here? https://github.com/bcgov/EDUC-EDX/blob/feature/edx-10/test/src/test_cases/edx-schools-frontend/test-school-inbox.js

File: frontend/src/components/SecureExchange/AccessUsersPage.vue (hunk 2 above)
Comment (cditcher) [reply frontend/src/components/SecureExchange/AccessUsersPage.vue]: Same with this.... am I losing it? https://github.com/bcgov/EDUC-EDX/blob/feature/edx-10/frontend/src/components/SecureExchange/AccessUsersPage.vue

**Pull request closed without merging**